test_app/tests.py

```diff
@@ -538,13 +538,13 @@ def _parse_to(to):
                 if not name:
                     ret.append(address)
                 else:
-                    ret.append("%s <%s>" % (name, address))
+                    ret.append("\"%s\" <%s>" % (name, address))
```

Comment:
Just food for thought: if we replicate the same logic in the tests as the one we have in the code, doesn't it mean that both can make the same mistakes?